homeassistant/components/meraki/device_tracker.py

```diff
@@ -123,10 +126,14 @@ def _handle(self, hass, data):
                 attrs["seenTime"] = i["seenTime"]
             if i.get("ssid", False):
                 attrs["ssid"] = i["ssid"]
+            if ap_mac:
+                attrs["apMac"] = ap_mac
```

Comment:
```suggestion
                attrs["ap_mac"] = ap_mac
```

Reply:
Isn't the access point mac address static information? Why do we need that in the state attributes?

Reply:
No the access point Is not static. When a devices moves between APs the AP's mac address will update